custom-metrics-api/custom-metrics-apiserver-deployment.yaml

```diff
@@ -19,6 +19,11 @@ spec:
       serviceAccountName: custom-metrics-apiserver
       containers:
       - name: custom-metrics-apiserver
+        resources:
```

Comment:
We've talked about moving over to helm, I guess it would make sense to template these values then. Right now this'll work for our use case, with the machine setup we use. Perhaps a note in the README.md about these values?

Reply:
Done